src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -22,6 +22,14 @@
 @EnableConfigurationProperties
 @ConfigurationProperties(prefix = "datarepo")
 public class ApplicationConfiguration {
+    /**
+     * Adding a static variable that can be accessed from the testing code since the MockitoJUnitRunner doesn't seem
+     * to allow you to autowire and having the same object mapper used in the tests is important
```

Comment:
Do you need the same object, or would it be OK to use another object that was created using the same registered modules?

If you have a test that doesn't use injection, you can create same object mapper by using `new ApplicationConfiguration(). objectMapper()`.

Reply:
Ah fair point...I just wanted to make sure that there was a common way to create an ObjectMapper with the same config.  It feels roundabout to get an object mapper that way, but happy to change it